pom.xml

```diff
@@ -41,26 +41,22 @@
     <!-- Build plugin versions -->
     <vertx.plugin.version>1.0.23</vertx.plugin.version>
     <freelib.maven.version>0.1.2</freelib.maven.version>
+    <!-- (running with Failsafe 3.0.0-M5 causes an IllegalAccessError during the integration-test phase) -->
+    <maven.failsafe.plugin.version>2.18.1</maven.failsafe.plugin.version>
 
     <!-- Test dependency versions -->
-    <testcontainers.version>1.15.2</testcontainers.version>
+    <docker.plugin.version>0.36.1</docker.plugin.version>
 
     <!-- Name of the main Vert.x verticle -->
     <main.verticle>edu.ucla.library.avpairtree.verticles.MainVerticle</main.verticle>
 
     <!-- Testing properties -->
     <live.test.reloads>false</live.test.reloads>
+    <localstack.edge.port>4566</localstack.edge.port>
```

Comment:
The parent project creates a ${http.port} for this purpose: https://github.com/ksclarke/freelib-build-tools/blob/6c60e5349d23d58be5d2432d1c98e974f3959e1a/freelib-parent/pom.xml#L208

But you could use that same plugin with a different variable name if you want the name to be specific to localstack.

Reply:
sure

Reply:
What's the difference between http.port and http.test.port in how you intend them to be used?

Reply:
Tbh, I don't really remember. I think it was originally because I needed an additional port for a project that needed two ports open during testing. In practice, more recently, I've just used ${http.port} for testing. It would probably make more sense if I was using ${http.test.port} though.